Automattic/wp-calypso
# Add AI Assistant pricing card

## Proposed Changes

* This diff adds product information for the AI Assistant product and adds the product to show on https://cloud.jetpack.com/pricing

## Testing Instructions

* Checkout this branch locally 
* Start calypso and the Jetpack Cloud environment with `yarn & yarn start-jetpack-cloud-p`
* Navigate to http://jetpack.cloud.localhost:3001/pricing
* Confirm that you see the "AI Assistant" card in the products list
![Screenshot 2023-06-01 at 5 10 34 PM](https://github.com/Automattic/wp-calypso/assets/18016357/75222345-a8ff-4a55-a259-acb25cb0f3b2)

* The lightbox should work when clicking "More About AI Assistant"
![Screenshot 2023-06-01 at 5 10 40 PM](https://github.com/Automattic/wp-calypso/assets/18016357/71bf746f-2fa9-4296-8492-e0092e710600)

* Confirm that clicking "Get" adds the correct product to your cart
* Check that the copy looks correct and makes sense

## Pre-merge Checklist
